[bug] tests broken for tabs

Wasn't quite finished working on fixing tabs tests when a PR was merged.  `main` branch now has the broken test and ❌ on test build.

Will close this issue with a new PR.

```
elements/pf-tabs/test/pf-tabs.spec.ts:159:14 - error TS18048: 'snapshot.children' is possibly 'undefined'.

159       expect(snapshot.children.find(x => x.role === 'tabpanel')?.name).to.equal('Database');
                 ~~~~~~~~~~~~~~~~~

elements/pf-tabs/test/pf-tabs.spec.ts:175:13 - error TS2488: Type 'A11yTreeSnapshot[] | undefined' must have a '[Symbol.iterator]()' method that returns an iterator.

175       const [disabledTab] = snapshot.children;
                ~~~~~~~~~~~~~

elements/pf-tabs/test/pf-tabs.spec.ts:176:51 - error TS7006: Parameter 'x' implicitly has an 'any' type.

176       const tabButton = disabledTab.children.find(x => x.role === 'button')!;
```

--- a/elements/pf-tabs/test/pf-tabs.spec.ts
+++ b/elements/pf-tabs/test/pf-tabs.spec.ts
@@ -156,7 +156,7 @@ describe('<pf-tabs>', function() {
 
     it('should activate its panel', function() {
       expect(panel.hasAttribute('hidden')).to.be.false;
-      expect(snapshot.children.find(x => x.role === 'tabpanel')?.name).to.equal('Database');
+      expect(snapshot.children?.find(x => x.role === 'tabpanel')?.name).to.equal('Database');
     });
   });
 
@@ -172,8 +172,8 @@ describe('<pf-tabs>', function() {
     });
 
     it('should disable the button', function() {
-      const [disabledTab] = snapshot.children;
-      const tabButton = disabledTab.children.find(x => x.role === 'button')!;
+      const [disabledTab] = snapshot.children!;
+      const tabButton = disabledTab.children!.find(x => x.role === 'button')!;
       expect(tabButton.disabled).to.equal(true);
     });
 

